lib/govuk_publishing_components/presenters/breadcrumbs.rb

```diff
@@ -21,7 +21,7 @@ def structured_data
       def item_list_element
```

Comment:
`request_path` is referenced elsewhere in this file - I think we can remove all these as it's not used any more.

Reply:
Removed `request_path`